src/daos/rinkeby.json

```diff
@@ -50,5 +50,88 @@
         }
       ]
     }
+  },
+  {
+    "address": "0x85017d3aeef46411dfb41f13ecd547cbc809d1f0",
+    "name": "1Hive Border Cases",
+    "description": "Building an experimenting with decentralized applications and protocols with an effort to build a sustainable and thriving economy around a currency called Honey which is used to influence the direction of the community",
+    "token":{
+      "logo": "https://raw.githubusercontent.com/1Hive/dao-list/master/src/assets/rinkeby/0x7777cd7c9c6d3537244871ac8e73b3cb9710d45a/nativeToken.svg"
```

Comment:
we need to create a new folder under assets with this new dao address

Reply:
done, also included same assets

Reply:
@famole you missed what we talked on discord about having, a test with a regex to avoid svg being uploaded and we should remove the currents svg files